service/artifacts/Config.toml

```diff
@@ -1,17 +1,17 @@
-modules_dir = "artifacts/"
```

Comment:
I think that's a wrong file to use for distro crate. You probably want configs from `builtin-package` directory.

Also, I think `builtin-package` dir should be removed after full migration to crates.

Reply:
It seems that this file was used to run it in `mrepl`... Not sure if it's relevant right now.

If no docs mention `mrepl`, then we probably can remove all these local configs. And create an issue to migrate this probject to flox.

@boneyard93501 wdyt?

Reply:
The configs from `builtin-package` aren't in the toml format used in marine. Also, there are two configs for each module in `builtin-package`, whereas marine `Config.toml` describes the whole service in a single file.

Also, I don't think it's a good idea to use separate configs for each module, `Config.toml` should be the standard for service deployment everywhere.